usr/share/device-quirks/scripts/ayaneo/air_plus/air_plus_dsdt.sh

```diff
@@ -5,14 +5,19 @@ if [ $(whoami) != 'root' ]; then
    exit 1
 fi
 
-mkdir -p /run/media/boot
-mount -L frzr_efi /run/media/boot
+if ! grep -q 'acpi_override' /boot/loader/entries/frzr.conf
```

Comment:
I think we should still call acpi_override by model. If you were to move this drive into a device with another AMI BIOS this would need to be cleaned up.

Reply:
Sure, I'll adjust small things like this once everything is working. To make it easier on my brain I'm naming things by what it is right now.